index.html

```diff
@@ -0,0 +1,172 @@
+                    </div>
+                    </div> <!-- /form-group -->
+                    <div class="form-group">
+                      <div class="row">
+                        <div class="row">
+                          <label for="inputPasswordConfirm" class="control-label col-xs-offset-1 col-sm-offset-1 col-md-offset-1 col-lg-offset-1">Confirma tu contraseña</label>
+                        </div>
+                        <div class="row">
+                          <input class="col-xs-offset-2 col-sm-offset-2 col-md-offset-2 col-lg-offset-2 col-sm-8 col-xs-8 col-lg-8 col-md-8" type="password" class="form-control" id="inputPasswordConfirm" data-match="#inputPassReg" data-match-error="Ups, las contraseñas no coinciden." data-error="Completa este campo" placeholder="Confirma tu contraseña" required>
+                        </div>
+                        <div class="row">
+                          <div class="col-xs-offset-2 col-sm-offset-2 col-md-offset-2 col-lg-offset-2 col-sm-8 col-xs-8 col-lg-8 col-md-8 help-block with-errors"></div>
+                        </div>
+                      </div>
+                      </div> <!-- /form-group -->
+                      <div class="form-group">
+                        <button id="register-btn" type="submit" data-dismiss="modal" class=" mb-1 btn btn-primary col-md-offset-3 col-lg-offset-3 col-md-6 col-lg-6 col-sm-offset-2 col-xs-offset-2 col-sm-8 col-xs-8 btn-registro">Enviar</button>
+                      </div>
+                    </form>
+                    <div class="modal-footer">
+                      <button type="button" class="btn btn-default" data-dismiss="modal">Cerrar</button>
+                    </div>
+                    </div> <!-- /modal-body -->
+                    </div> <!-- /modal-content -->
+                    </div> <!-- /modal-dialog -->
+                    </div><!-- /modal fade - fin modal de registro -->
+                    <!-- Modal Log In -->
+                    <div class="modal fade" id="modalLog" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+                      <div class="modal-dialog" role="document">
+                        <div class="modal-content">
+                          <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                            <h3 class="text-center">Iniciar sesión</h3>
+                          </div>
+                          <div class="modal-body">
+                            <form data-toggle="validator" role="form">
+                              <div class="row">
+                                <div class="form-group">
+                                  <div class="row">
+                                    <label for="inputEmail" class="control-label col-xs-offset-1 col-sm-offset-1 col-md-offset-1 col-lg-offset-1">Correo electrónico</label>
```

Comment:
Creo que aquí debería ser for="inputEmailLog", y en el label del otro modal for="inputEmailReg" ?

Y lo mismo para los otros inputs, como el password.